src/test/scala/org/apache/spark/sql/delta/OptimisticTransactionSuite.scala

```diff
@@ -60,8 +60,7 @@ class OptimisticTransactionSuite extends QueryTest with SharedSparkSession {
       txn.filterFiles()
       val winningTxn = log.startTransaction()
       winningTxn.commit(addA :: Nil, ManualUpdate)
-      // TODO: intercept a more specific exception
-      intercept[DeltaConcurrentModificationException] {
```

Comment:
Sorry for the late reply. Could you resolve the conflict? The test has been moved to https://github.com/delta-io/delta/blob/master/core/src/test/scala/org/apache/spark/sql/delta/OptimisticTransactionLegacyTests.scala#L67 . Otherwise, LGTM